CMakeLists.txt

```diff
@@ -1,10 +1,7 @@
 cmake_minimum_required(VERSION 3.0)
 project(NLCL)
 
-add_library(NLCL forest.cpp)
-add_executable(testForest main.cpp)
+include_directories(NLCL include)
```

Comment:
It's better to use target_include_directories(NLCL PUBLIC include). You can find a reference in ULP repository.